examples/rsp-next-ts/pages/index.tsx

```diff
@@ -0,0 +1,394 @@
+import Head from "next/head";
+import styles from "../styles/Home.module.css";
+import React from "react";
+import {
+  ActionMenu,
+  Item,
+  Menu,
+  MenuTrigger,
+  ListView,
+  Calendar,
+  DateField,
+  DatePicker,
+  DateRangePicker,
+  RangeCalendar,
+  TimeField,
+  Badge,
+  LabeledValue,
+  Picker,
+  ActionButton,
+  Breadcrumbs,
+  Button,
+  ButtonGroup,
+  Cell,
+  Checkbox,
+  CheckboxGroup,
+  Column,
+  Divider,
+  Flex,
+  Form,
+  Grid,
+  Heading,
+  Image,
+  Link,
+  ListBox,
+  LogicButton,
+  NumberField,
+  Radio,
+  RadioGroup,
+  Row,
+  SearchField,
+  Switch,
+  TableBody,
+  TableHeader,
+  TableView,
+  TabList,
+  TabPanels,
+  Tabs,
+  Text,
+  TextArea,
+  TextField,
+  ToggleButton,
+  DialogTrigger,
+  AlertDialog,
+  ContextualHelp,
+  Content,
+  Dialog,
+  Header,
+  TooltipTrigger,
+  Tooltip,
+  ComboBox,
+  RangeSlider,
+  Slider,
+  Meter,
+  ProgressBar,
+  ProgressCircle,
+  StatusLight,
+  Footer,
+  IllustratedMessage,
+  Keyboard,
+  View,
+  Well,
+} from "@adobe/react-spectrum";
+import Edit from "@spectrum-icons/workflow/Edit";
+import NotFound from "@spectrum-icons/illustrations/NotFound";
+import Section from "../components/Section";
+
+export default function Home() {
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>React Spectrum + NextJS</title>
+        <meta name="description" content="React Spectrum + NextJS" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main className={styles.main}>
+        <Heading level={1}>
+          React Spectrum +{" "}
+          <Link>
+            <a href="https://nextjs.org">Next.js</a>
+          </Link>
+        </Heading>
+
+        <Flex direction="column" gap="size-125">
+          <Section title="Buttons">
+            <ButtonGroup>
+              <Button variant="primary">Primary</Button>
+              <Button variant="secondary">Secondary</Button>
+              <ActionButton>
+                <Edit />
+                <Text>Action Button</Text>
+              </ActionButton>
+              <LogicButton variant="and">Logic Button</LogicButton>
+              <ToggleButton>Toggle Button</ToggleButton>
+            </ButtonGroup>
+          </Section>
+
+          <Section title="Collections">
+            <ActionMenu>
+              <Item>Action Menu Item 1</Item>
+              <Item>Action Menu Item 2</Item>
+              <Item>Action Menu Item 3</Item>
+            </ActionMenu>
+            <ListBox width="size-2400" aria-label="Listbox">
+              <Item>ListBox Item 1</Item>
+              <Item>ListBox Item 2</Item>
+              <Item>ListBox Item 3</Item>
+            </ListBox>
+            <ListView
+              selectionMode="multiple"
+              aria-label="Static ListView items example"
+              maxWidth="size-6000"
+            >
```

Comment:
Does this project follow a different linter since these on on new lines?

Reply:
It has it's own eslint config (the default provided by `create-next-app`) and it was formatted with prettier, which I think is fine for this example IMO.